### DISCUSSION TITLE
We are back !

### DISCUSSION THREAD

**Discussion by heheyas** [gsgen3d/gsgen#1] [Announcements]
We were accidentally suspended by the GitHub. Now we are back ! The project page is available at https://gsgen3d.github.io/.

**Comment by SoftologyPro** [comment]
Cannot git clone...

git clone https://github.com/gsgen3d/gsgen
Cloning into 'gsgen'...
remote: Enumerating objects: 219, done.
remote: Counting objects: 100% (219/219), done.
remote: Compressing objects: 100% (186/186), done.
Receiving objects:  95% (209/219), 4.39 MiB | 2.91 MiB/sremote: Total 219 (delta 32), reused 212 (delta 28), pack-reusedReceiving objects: 100% (219/219), 4.92 MiB | 2.95 MiB/s, done.

Resolving deltas: 100% (32/32), done.
error: invalid path 'conf/aux/pe_normal.yaml'
fatal: unable to checkout working tree
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry with 'git restore --source=HEAD :/'

**Reply by heheyas** [reply]
Hi, SoftologyPro

Thank you for your interest! Are you cloning on a Windows computer? If so, you can try:
```
git config core.protectNTFS false
```
as suggested in https://stackoverflow.com/questions/63727594/github-git-checkout-returns-error-invalid-path-on-windows.

**Reply by SoftologyPro** [reply]
Yes, Windows.  I cannot run that command before checking out.  Gives "fatal: not in a git directory".
I tried to download and extract the zip of the repo and get this.

```
! Attempting to correct the invalid file or folder name
! Renaming D:\Tests\gsgen-main\gsgen-main\conf\aux to D:\Tests\gsgen-main\gsgen-main\conf\_aux
! Attempting to correct the invalid file or folder name
! Renaming D:\Tests\gsgen-main\gsgen-main\conf\aux\pe_normal.yaml to D:\Tests\gsgen-main\gsgen-main\conf\_aux\pe_normal.yaml
! Attempting to correct the invalid file or folder name
! Renaming D:\Tests\gsgen-main\gsgen-main\conf\aux\point_e.yaml to D:\Tests\gsgen-main\gsgen-main\conf\_aux\point_e.yaml
```

Something corrupted there.

**Reply by SoftologyPro** [reply]
On Windows "aux" is a reserved name you cannot use for a file and/or directory name.
So to fix for Windows, is it possible to change the aux directory name?
